src/main/kotlin/com/exactpro/th2/rptdataprovider/entities/responses/MessageInfoWrapper.kt

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.exactpro.th2.rptdataprovider.entities.responses
+
+data class MessageInfoWrapper(val message: Message, val error: String? = null)
+
+data class MessageGRPCInfoWrapper(val message: com.exactpro.th2.common.grpc.Message?, val error: String? = null)
```

Comment:
Not sure if we really need both of them

Reply:
Also, let's rename these classes
MessageWrapper, SearchResult, or something similar